service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -0,0 +1,455 @@
+[id="upgrading-ossm"]
+= Upgrading {ProductName} from version 1.1 to version 2.0
+include::modules/ossm-document-attributes.adoc[]
+:context: upgrading-ossm
+toc::[]
+
+To access the most current features of {ProductName}, upgrade to the most current version, 2.0. Upgrading from version 1.1 to 2.0 requires manual steps that migrate your workloads and apps to a new instance of {ProductName} running the new version.
+
+.Prerequisites
+
+* You must upgrade to {product-title} 4.6. before you upgrade to {ProductName} 2.0.
+* You must have an updated {ProductName} operator. If you selected the *automatic* upgrade path, the operator automatically downloads the latest information. However, there are steps you must take to use the features in {ProductName} version 2.0.
+
+[id="ossm-upgrading-upgrade_{context}"]
+== Upgrading {ProductName}
+
+To upgrade {ProductName}, you must create an instance of {ProductName} 2.0 in a new namespace. Then, once it's configured, move your microservice applications and workloads from your old mesh to the new service mesh.
+
+.Procedure
+
+. Back up your control plane configuration. Switch to the project that contains your `ServiceMeshControlPlane` resource.
++
+[source,terminal]
+----
+oc project istio-system
+----
++
+. Enter the following command to retrieve the current configuration. Your <smcp_name> is specified in the metadata of your `ServiceMeshControlPlane` resource, for example `basic-install` or `full-install`.
++
+[source,terminal]
+----
+$ oc get servicemeshcontrolplanes.v1.maistra.io <smcp_name> -o yaml > <smcp_name>.v1.yaml
+----
++
+. Convert your `ServiceMeshControlPlane` to a v2 control plane version that contains information about your configuration as a starting point.
++
+[source,terminal]
+----
+$ oc get smcp <smcp_name>
+----
++
+. Create a project. In the {product-title} console Project menu, click `New Project` and enter a name for your project, `istio-system-upgrade`, for example. Or, you can run this command from the CLI.
++
+[source,terminal]
+----
+$ oc new-project istio-system-upgrade
+----
++
+. Create a `ServiceMeshControlPlane` in the new namespace. On the command line, run the following command to deploy the control plane with the v2 version of the `ServiceMeshControlPlane` that you retrieved. In this example, replace `<V2_SMCP> `with the path to your file.
++
+[source,terminal]
+----
+$ oc create -n istio-system-update -f <V2_SMCP>.yaml
```

Comment:
Would you use `<smcp_name>.v2.yaml` for consistency?  Also, prior to deployment, the user should modify the `spec.version` field to `v2.0` and should modify the `metadata.namespace` field to the new namespace, e.g. `istio-system-upgrade`.